File: 02. CONV_UNI_SOAP_DOTNET_GUARDERAS/02. CLIESC/CLIESC/CLIESC/ec.edu.monster.controlador/ConvUniControlador.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CLIESC.ec.edu.monster.modelo;

namespace CLIESC.ec.edu.monster.controlador
{
    public class ConvUniControlador
    {
        private ConvUniModelo modelo;

        public ConvUniControlador()
        {
            modelo = new ConvUniModelo();
        }

        public object convertCF(double c)
        {
            return Convert.ToDouble(modelo.convertCF(c));
        }

        public object convertFC(double f)
        {
            return Convert.ToDouble(modelo.convertFC(f));
        }


    }
}

File: 02. CONV_UNI_SOAP_DOTNET_GUARDERAS/02. CLIMOV/WS_CONVUNI_SOAPDOTNET_CLIMOV/app/src/main/java/ec/edu/monster/controlador/ConvUniControlador.java
package ec.edu.monster.ws_convuni_soapdotnet_climov.controlador;

import ec.edu.monster.ws_convuni_soapdotnet_climov.modelo.ConvUniModelo;

public class ConvUniControlador {

    private ConvUniModelo modelo;

    public ConvUniControlador(){
        modelo = new ConvUniModelo();
    }

    public String convertCF(String c){
        double envio = Double.valueOf(c);

        double recibo = modelo.convertCF(envio);

        String respuesta = "";

        if(recibo == -1.0){
            respuesta = "error";
        }else{
            respuesta = Double.toString(recibo);
        }

        return respuesta;
    }

    public String convertFC(String f){
        double envio = Double.valueOf(f);

        double recibo= modelo.convertFC(envio);

        String respuesta = "";

        if(recibo == -1.0){
            respuesta = "error";
        }else{
            respuesta = Double.toString(recibo);
        }

        return respuesta;
    }

}
